### ISSUE TITLE
Traditional queries no longer work witth new release

### ISSUE THREAD

**Issue by starstuck** [dalmatinerdb/dqe#3]
I have now support of new queries, like

```
SELECT 'a'.'b'.* BUCKET 'x' LAST 6os
```

but when I try providing full path

```
SELECT 'a'.'b'.'c' BUCKET 'x' LAST 6os
```

I get errors. Crash logs says:

```
Error in process <0.245.0> on node 'dalmatinerfe@127.0.0.1' with exit value:
{[{reason,function_clause},{mfa,{dalmatiner_idx_handler,handle,2}},{stacktrace,[{dqe,needs_buckets,[{calc,[],{get,{<<"565c43872c136ca11e786758">>,[<<"c3bcee12-0680-4bf3-8237-f51b48330dd8">>,<<"base">>,<<"cpu">>,<<"idle">>]}}},[]],[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_checkouts/dqe/src/dqe.erl"},{line,421}]},{lists,foldl,3,[{file,"lists.erl"},{line,1262}]},{dqe,prepare,1,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_checkouts/dqe/src/dqe.erl"},{line,124}]},{dqe,run,2,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_checkouts/dqe/src/dqe.erl"},{line,83}]},{timer,tc,3,[{file,"timer.erl"},{line,197}]},{dalmatiner_idx_handler,handle,2,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_build/default/lib/dalmatiner_frontend/src/dalmatiner_idx_handler.erl"},{line,28}]},{cowboy_handler,handler_handle,4,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_build/default/lib/cowboy/src/cowboy_handler.erl"},{line,111}]},{cowboy_protocol,execute,4,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_build/default/lib/cowboy/src/cowboy_protocol.erl"},{line,442}]}]},{req,[{socket,#Port<0.2190>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.245.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},56363}},{host,<<"localhost">>},{host_info,undefined},{port,8081},{path,<<"/">>},{path_info,undefined},{qs,<<"q=SELECT%20%27c3bcee12-0680-4bf3-8237-f51b48330dd8%27.%27base%27.%27cpu%27.%27idle%27%20BUCKET%20%27565c43872c136ca11e786758%27%20LAST%2060s">>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8081">>},{<<"user-agent">>,<<"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:42.0) Gecko/20100101 Firefox/42.0">>},{<<"accept">>,<<"application/x-msgpack">>},{<<"accept-language">>,<<"en-GB,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"referer">>,<<"http://localhost:8081/?query=SELECT%20%27c3bcee12-0680-4bf3-8237-f51b48330dd8%27.%27base%27.%27cpu%27.%27idle%27%20BUCKET%20%27565c43872c136ca11e786758%27%20LAST%2060s">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}],[{cowboy_protocol,execute,4,[{file,"/Users/Szarsti/wsp/dalmatiner-fe/_build/default/lib/cowboy/src/cowboy_protocol.erl"},{line,442}]}]}
```

I am looking into potential fix.

**Comment by starstuck** [comment]
I am running frontend against dqe v0.1.27

**Comment by Licenser** [comment]
good catch, thanks! Fixed in 0.1.28
